New serverless pattern - Dynamic Bedrock model selection with API Gateway and Lambda (#1750)
aws-samples/serverless-patterns: dynamic-bedrock-svls-sam/README.md

> # Dynamic Bedrock model selection with API Gateway and Lambda

Reviewer: ```suggestion
# Dynamic Amazon Bedrock model selection with API Gateway and Lambda
```

> This pattern creates an API Gateway REST API with Lambda proxy integration which places invoke calls to Amazon Bedrock models for text and image generation.

Reviewer: ```suggestion
This pattern creates an API Gateway REST API with Lambda proxy integration which invokes Amazon Bedrock models for text and image generation.
```

> Learn more about this pattern at Serverless Land Patterns: <TBD>

Reviewer: ```suggestion
Learn more about this pattern at Serverless Land Patterns: https://serverlessland.com/patterns/dynamic-bedrock-svls-sam
```
